webroot/results/misc/evolution/update7205.php

```diff
@@ -33,7 +33,23 @@
 echo "<p></p>";
 
 #--- Define what to show in what order and with nice short names
-$whats = explode( ' ', '333:3x3 444:4x4 555:5x5 222:2x2 333bf:bld 333oh:hand 333fm:moves:1 333ft:feet minx:mega pyram:pyra sq1:sq1 clock:clock 666:6x6 777:7x7 magic:magic mmagic:master 444bf:4bld 555bf:5bld 333mbf:multi:1' );
+$events = dbQueryHandle("
+      SELECT id, name
+      FROM Events
+      WHERE rank < 900
+      ORDER BY rank
+    ");
+
+$whats = [];
+while($row = mysql_fetch_row($events)) {
+  $new_item = $row[0] . ":" . $row[1];
```

Comment:
nit: I'd have written this a little more verbosely, because I like the convention from other parts of the code. See [here](https://github.com/thewca/worldcubeassociation.org/blob/8a53d2f1849f23631fa03ded7474fddab8c28966/webroot/results/admin/check_regional_record_markers.php#L141) for an example. Something like this:

```php
list( $id, $name ) = $row;
$new_item = $id . ":" . $name;
```

And then you can use `$id` below, rather than the harder to read `$row[0]`